Dockerfile

```
@@ -19,28 +23,96 @@ COPY ./app/img/ /source/app/img/
 COPY ./app/ts/ /source/app/ts/
 RUN npm run build
 
+# --------------------------------------------------------
+# Base Image: Create the base image that will host the app
+# --------------------------------------------------------
+
 # Cache the kubo image
 FROM ipfs/kubo:v0.25.0@sha256:0c17b91cab8ada485f253e204236b712d0965f3d463cb5b60639ddd2291e7c52 as ipfs-kubo
 
 # Create the base image
-FROM debian:12.2-slim@sha256:93ff361288a7c365614a5791efa3633ce4224542afb6b53a1790330a8e52fc7d
+FROM debian:12.2-slim@sha256:93ff361288a7c365614a5791efa3633ce4224542afb6b53a1790330a8e52fc7d as base
```

No need for the named image anymore.
```suggestion
FROM debian:12.2-slim@sha256:93ff361288a7c365614a5791efa3633ce4224542afb6b53a1790330a8e52fc7d
```